.evergreen/config.in.yml

```diff
@@ -148,43 +148,21 @@ functions:
             export AWS_REGION='${AWS_REGION}'
             export AWS_CMK_ID='${AWS_CMK_ID}'
             export AWS_DEFAULT_REGION='us-east-1'
-            export KMIP_TLS_CA_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem"
-            export KMIP_TLS_CERT_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem"
+            export CSFLE_TLS_CA_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem"
+            export CSFLE_TLS_CLIENT_CERT_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem"
           EOT
           fi
-    - command: shell.exec
+    - command: subprocess.exec
       type: test
       params:
+        env:
+          TEST_CSFLE: "true"
+        add_expansions_to_env: true
         working_dir: "src"
         timeout_secs: 300
-        shell: bash
-        script: |
-          ${PREPARE_SHELL}
-
-          if [ -n "${CLIENT_ENCRYPTION}" ]; then
-            # Disable xtrace (just in case it was accidentally set).
-            set +x
-            . ./prepare_client_encryption.sh
-            rm -f ./prepare_client_encryption.sh
-          fi
-
-          export VERSION=${VERSION}
-          export DRIVERS_TOOLS=${DRIVERS_TOOLS}
-
-          if [ -z "${RUN_WITH_MONGOCRYPTD}" ]; then
-            # Set up crypt shared lib if we don't want to use mongocryptd
-            source ${PROJECT_DIRECTORY}/.evergreen/prepare-crypt-shared-lib.sh
-            echo "CRYPT_SHARED_LIB_PATH: $CRYPT_SHARED_LIB_PATH"
-          else
-            echo "CRYPT_SHARED_LIB_PATH not set; using mongocryptd"
-          fi
-
-          TEST_NPM_SCRIPT="${TEST_NPM_SCRIPT|check:integration-coverage}" \
-          MONGODB_URI="${MONGODB_URI}" \
-          AUTH=${AUTH} SSL=${SSL} TEST_CSFLE=true \
-          MONGODB_API_VERSION="${MONGODB_API_VERSION}" \
-          SKIP_DEPS=${SKIP_DEPS|1} \
-            bash ${PROJECT_DIRECTORY}/.evergreen/run-tests.sh
+        binary: bash
+        args: 
+          - .evergreen/run-tests.sh
```

Comment:
Half of this block was configuring FLE variables, the other half re-exported expansions.  It can all be removed.